File: Application/server/src/test/resources/PlagiarismChecker-Data/Python Samples/set01/ConverterX.py
import NumberStringConstants

class ConverterX(object):

    numberString=""
    number=""
    numNames = NumberStringConstants.digits
    tensNames = NumberStringConstants.tens

    def convertToWords(self, number) :

        ste=""

        if number % 100 < 20:
            ste = self.numNames[int(number % 100)]
            number /= 100
        else:
            ste = self.numNames[int(number % 10)]
            number /= 10
            ste = self.tensNames[int(number % 10)] + ste
            number /= 10

        if number == 0:

            return ste

        else:

           return self.numNames[int(number)] + " hundred" + ste

File: Application/server/src/test/resources/PlagiarismChecker-Data/set07/Sample2/src/src/ConverterX.java
package src;
import src.NumberStringConstants;;

public class ConverterX {
	
	String numberString;
	int number;
	static String[] numNames = NumberStringConstants.numNames;
	static String[] tensNames = NumberStringConstants.tensNames;
	
	static String convertToWords(int number) {
	    String soFar;

	    if (number % 100 < 20){
	      soFar = numNames[number % 100];
	      number /= 100;
	    }
	    else {
	      soFar = numNames[number % 10];
	      number /= 10;

	      soFar = tensNames[number % 10] + soFar;
	      number /= 10;
	    }
	    if (number == 0) return soFar;
	    return numNames[number] + " hundred" + soFar;
	  }
	
}
